psc-package/Main.hs

```diff
@@ -224,6 +224,27 @@ listSourcePaths = do
   paths <- getSourcePaths pkg db depends
   traverse_ (echo . pathToTextUnsafe) paths
 
+verifyPackageSet :: IO ()
+verifyPackageSet  = do
+  -- Clone all repos into the packages/ directory
+  ps <- readPackageFile 
+  paths <- fromList <$>
+    (for (getGitRepoList ps) (\(name, (repo, version)) -> do
+      let pkgDir = ".psc-package" </> fromText name </> fromText version
```

Comment:
Let's use the usual path here: `.psc-package/<set name>/<package name>/<version>`